spec/example-unnamed-annotation.ttl

```diff
@@ -0,0 +1,4 @@
+PREFIX prov: <http://www.w3.org/ns/prov#>
```

Comment:
I suggest using `@prefix` instead of `PREFIX` to make it consistent with other examples that are closed located.

Reply:
Consistency is good -- let's migrate to `PREFIX`. It's been awhile since RDF 1.1.

Reply:
Yes, I thought we decided to to that in all specs? (My _personal_ preference is `prefix`, but going for consistency.)